README.md

```diff
@@ -12,6 +13,22 @@ Small movie site to browse movies:
 Try the demo at
 [https://awnigharbia.github.io/filmy/#/](https://awnigharbia.github.io/filmy/#/)
 
+## Setup API KEY
+
+First you need to register an account on
+[themoviedb.org](https://www.themoviedb.org/).
+
+After registration you can find your api key at
+[https://www.themoviedb.org/settings/api](https://www.themoviedb.org/settings/api).
+
+Next, create an `.env` file in the root directory of your project and add the
+following lines:
+
+```js
+VITE_API_KEY=<your_api_key>
+VITE_API_URL=<your_api_url> (default: https://api.themoviedb.org/3)
```

Comment:
```suggestion
VITE_API_URL=<your_api_url>
```

I suggest adding the text  "The default value of `VITE_API_URL` is `https://api.themoviedb.org/3`" outside of the code block. It might confuse new users thinking that it belongs there.